src/components/file/examples/file.tsx

```diff
@@ -71,4 +80,9 @@ export class FileExample {
         event.stopPropagation();
         this.readonly = !!event.detail;
     };
+
+    private setInvalid = (event: CustomEvent<boolean>) => {
+        event.stopPropagation();
+        this.invalid = !!event.detail;
```

Comment:
`event.detail` is already boolean
```suggestion
        this.invalid = event.detail;
```